packages/babel-plugin-jsx-dom-expressions/test/__dom_fixtures__/attributeExpressions/output.js

```diff
@@ -224,25 +221,20 @@ const template20 = (() => {
   _$addEventListener(_el$27, "input", doSomething, true);
   _$addEventListener(_el$28, "input", doSomethingElse, true);
   _el$28.readOnly = value;
-  _$effect(
-    _p$ => {
-      const _v$4 = min(),
-        _v$5 = max(),
-        _v$6 = min(),
-        _v$7 = max();
-      _v$4 !== _p$._v$4 && _$setAttribute(_el$27, "min", (_p$._v$4 = _v$4));
-      _v$5 !== _p$._v$5 && _$setAttribute(_el$27, "max", (_p$._v$5 = _v$5));
-      _v$6 !== _p$._v$6 && _$setAttribute(_el$28, "min", (_p$._v$6 = _v$6));
-      _v$7 !== _p$._v$7 && _$setAttribute(_el$28, "max", (_p$._v$7 = _v$7));
-      return _p$;
-    },
-    {
-      _v$4: undefined,
-      _v$5: undefined,
-      _v$6: undefined,
-      _v$7: undefined
+  _$effect(() => {
+    if (_v$4 !== (_v$4 = min())) {
+      _$setAttribute(_el$27, "min", _v$4);
     }
-  );
+    if (_v$5 !== (_v$5 = max())) {
+      _$setAttribute(_el$27, "max", _v$5);
+    }
+    if (_v$6 !== (_v$6 = min())) {
+      _$setAttribute(_el$28, "min", _v$6);
+    }
+    if (_v$7 !== (_v$7 = max())) {
+      _$setAttribute(_el$28, "max", _v$7);
+    }
```

Comment:
In a separate pull request I'd like to merge these 4 conditionals down into 2 conditionals, not exactly sure how that'd go though. I suppose it could be prone to side effects?